MCOL-3718 Use systemd instead of mysql-Columnstore

LinuxJedi opened the pull request:
MariaDB server is now managed using systemd instead of our own
mysql-Columnstore script. This means that MariaDB server can be stopped
/ started independently of ColumnStore

LinuxJedi committed:
MCOL-3718 Use systemd instead of mysql-Columnstore

MariaDB server is now managed using systemd instead of our own
mysql-Columnstore script. This means that MariaDB server can be stopped
/ started independently of ColumnStore

LinuxJedi commented:
This appears to break Buildbot because our RPMs don't appear to have MariaDB's systemd unit. I'm not sure why.

LinuxJedi committed:
Make mysqld work without systemd

ColumnStore should now start/stop mysqld where the systemd service is
not available.

benthompson15 reviewed (changes requested):
in void Oam::actionMysqlCalpont(MYSQLCALPONT_ACTION action)
check logic for MYSQL_STATUS 
before it was logging to mysql.status which is checked at end of function by
        if (checkLogStatus(mysqlStatus, "MySQL running"))
when that fails its set to MAN_OFFLINE and DEGRADED even though mysqld is running

LinuxJedi committed:
Fix restart hang

mcsadmin restart will do a status call. Before this would log status
output to a file which would be checked. If that file check failed a PID
check would be made.

We do not log the status to a file any more since it could have multiple
different outputs. The PID check could hit a race condition where it is
being checked before the process is up. This would mean there is no
module state change causing a hang.

This fix does the following:

1. Remove the file status check completely
2. Loop the PID check to give it time to come up
3. If the PID check fails drop to a DEGRADED state

This makes mcsadmin restart work correctly again.
